build.gradle

```diff
@@ -48,20 +49,6 @@ configurations {
     }
 }
 
-task cucumberCli() {
-    dependsOn assemble, testClasses
-    doLast {
-        javaexec {
-            main = "io.cucumber.core.cli.Main"
-            classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
-            args = [
-                    '--plugin', 'html:cucumber-report',
-                    '--plugin', 'json:cucumber.json',
-                    '--plugin', 'pretty',
-                    '--plugin', 'html:build/cucumber-report.html',
-                    '--plugin', 'json:build/cucumber-report.json',
-                    '--glue', 'org.mifos.integrationtest.cucumber',
-                    'src/test/java/resources']
```

Comment:
@danishjamal104 : Removing this has broken 3-4 features in the pipelines. html:build/cucumber-report.html and json:build/cucumber-report.json is coming as empty.  Found a workaround but this needs fixing.